### PR TITLE
Enhancement/3157 save idea info

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #3157

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: tests/phpunit/includes/TestCase.php [hunk 1]
```diff
@@ -248,6 +248,12 @@ protected function assertPostMetaExists( $post_id, $meta_key ) {
 		);
 	}
 
+	protected function assertPostMetaHasValue( $post_id, $meta_key, $meta_value ) {
+		$meta = $this->queryPostMeta( $post_id, $meta_key );
+		$this->assertNotNull( $meta );
+		$this->assertEquals( $meta_value, $meta['meta_value'], "Failted to assert that post $post_id has \"$meta_key\" meta with \"$meta_value\" value." );
```
Comment (tofumatt) [inline tests/phpunit/includes/TestCase.php:254]:
```suggestion
		$this->assertEquals( $meta_value, $meta['meta_value'], "Failed to assert that post $post_id has \"$meta_key\" meta with \"$meta_value\" value." );
```

Comment (tofumatt) [reply tests/phpunit/includes/TestCase.php:254]: Seems like a misspelling here, but then it _did_ pass. What's causing this to emit "failted"? I couldn't find it in our code, but is it worth noting why that… typo appears? 🤷🏻‍♂️

**Review by tofumatt — CHANGES_REQUESTED** [review]
Seems good, but I'm curious about a misspelling in the test case.

Also: can you add a QA brief? I know in this case it's essentially "Make sure tests pass", but better to leave a minimal one than an empty one 🙂

File: tests/phpunit/includes/TestCase.php (hunk 1 above)
Comment (eugene-manuilov) [reply tests/phpunit/includes/TestCase.php]: @tofumatt the last parameter is just a friendly message that will be displayed if the first two parameters don't equal.

**Review by tofumatt — APPROVED** [review]

**Pull request merged**